backend/public-graph/graph/resolver.go

```diff
@@ -2213,14 +2212,30 @@ func (r *Resolver) ProcessBackendPayloadImpl(ctx context.Context, sessionSecureI
 			ServiceVersion: v.Service.Version,
 		}
 
-		var mappedStackTrace *string
 		var structuredStackTrace []*privateModel.ErrorTrace
-		mappedStackTrace, structuredStackTrace, err = r.Store.EnhancedStackTrace(ctx, v.StackTrace, workspace, &project, errorToInsert, nil)
+		var stackFrameInput []*publicModel.StackFrameInput
 
+		if err := json.Unmarshal([]byte(v.StackTrace), &stackFrameInput); err == nil {
+			mapped, structured, err := r.getMappedStackTraceString(ctx, stackFrameInput, projectID, errorToInsert, pointy.String(fmt.Sprintf("%s-%s", v.Service.Name, v.Service.Version)))
```

Comment:
Just want to make this edge case won't cause issues - a lot of users leave this hardcoded to "git-sha" or an empty string when it isn't provided to the Highlight instance.

This could help promote users supplying their correct service version to the instance, but want to call out incase any one may be having issues with it working

Seems like worst case we would log the error and continue

Reply:
seems to me that now for accounts with an invalid git sha, we may get log spam from this happening and then the project's service would become unhealthy, but both seem fine. do you think there might be other issues with having this run by default?